networking/network_policy/creating-network-policy.adoc

```diff
@@ -1,7 +1,12 @@
 :_content-type: ASSEMBLY
 [id="creating-network-policy"]
 = Creating a network policy
+ifdef::openshift-enterprise[]
```

Comment:
The same comments apply here regarding the attribute files. Please see my preceding comment for details.